distroless/java missing libjpeg.so.62

<!-- Thanks for filing an issue! Please use this template to provide more context for the issue . -->

**Description of the issue**: The docker images build ok with Jib, but I get an exception thrown when the container runs. The application is a spring boot 2.1.x app that runs ok in the IDE against java 11.

**Expected behavior**: No exception

**Steps to reproduce**: Jib a spring boot 2.1.x app that uses image handling parts of java 11 and attempt to run the container on your local docker.

**Environment**: Elementary Linux / Java 11 / Gradle / Spring boot 2.1.x

**`jib-gradle-plugin` Configuration**: <!-- Delete this section if not used -->
```groovy
sourceCompatibility = JavaVersion.VERSION_11
targetCompatibility = JavaVersion.VERSION_11

bootRun {
  systemProperties = System.properties // needed for intellij
  systemProperties.remove("java.endorsed.dirs")
}

	jib {
		from.image 'gcr.io/distroless/java:11'
		to.image project.getParent().getName() + "/" + bootJar.getBaseName()
		container({
			useCurrentTimestamp = true
			ports = ['80']
			jvmFlags = ['-Djava.security.egd=file:/dev/./urandom', '-XX:+UseG1GC']
		})
		
	}
```

**Log output**: 

`org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'dataLoader': Invocation of init method failed; nested exception is java.lang.UnsatisfiedLinkError: /usr/lib/jvm/java-11-openjdk-amd64/lib/libjavajpeg.so: libjpeg.so.62: cannot open shared object file: No such file or directory`

`Caused by: java.lang.UnsatisfiedLinkError: /usr/lib/jvm/java-11-openjdk-amd64/lib/libjavajpeg.so: libjpeg.so.62: cannot open shared object file: No such file or directory`

--- a/WORKSPACE
+++ b/WORKSPACE
@@ -81,6 +81,7 @@ dpkg_list(
 
         #java
         "zlib1g",
+        "libjpeg62-turbo",
         "openjdk-8-jre-headless",
         "openjdk-11-jre-headless",
 

--- a/java/BUILD
+++ b/java/BUILD
@@ -15,6 +15,7 @@ cacerts_java(
     base = "//cc" if (not ("debug" in rule_name)) else "//cc:debug",
     debs = [
         packages["zlib1g"],
+        packages["libjpeg62-turbo"],
         packages[jre_deb],
     ],
     entrypoint = [

